client/src/components/Settings/index.tsx

```diff
@@ -10,46 +10,125 @@ import chevron from "icons/chevron.svg";
 import "./index.css";
 import DropDown from "components/Dropdown";
 import Text from "components/Text";
+import styled from "styled-components";
 
 const engineArray: Engine[] = ["youtube", "soundcloud"];
 
+const Button = styled.button`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  height: 2rem;
+  will-change: transform;
+  border: none;
+  text-decoration: none;
+  outline: none;
+  background-color: #393838;
+  font-weight: 500;
+  border-radius: 3px;
+  box-shadow: 1px 1px 2px black;
+  transition: box-shadow 0.2s;
+  margin-bottom: 1rem;
+
+  &:hover {
+    box-shadow: 2px 1px 5px black;
+  }
+`;
+
+const BackButton = styled(Button)`
+  & span {
+    width: 0;
+    transition: width 0.3s;
+  }
+
+  &:hover span {
+    width: 5rem;
+  }
+`;
+
 const Settings = () => {
   const [healthExpanded, setHealthExpanded] = useState<boolean>(false);
   const history = useHistory();
   return (
-    <div className="settings">
-      <button onClick={() => history.goBack()}>
+    <div
+      //className="settings"
```

Comment:
can be removed... feel free...

```suggestion
      //className="settings"
```